rules/style.js

```diff
@@ -107,9 +123,16 @@ module.exports = {
 		}],
 		// Require extra spaces inside objects
 		"object-curly-spacing": [2, "always"],
-		// Require quoting of object keys when necessary (e.g. var object = { "foo-bar": 5, bar: 2, foo: function() {}})
+		// Enforce operators at the end of a line except for ternary operators,
+		// which should be at the beginning of a line
+		"operator-linebreak": [2, "after",
+			{ "overrides": {"?": "before", ":": "before"}}
+		],
```

Comment:
I think we've agreed we want to put `&&` and `||` at the end of lines and ternary operators at the beginning, correct? Happy to change to our preference. This is what drives most of the changes.

Reply:
This will auto-format!